src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathScoring.h

```diff
@@ -183,7 +183,7 @@ namespace OpenMS
     void calculateDIAScores(OpenSwath::IMRMFeature* imrmfeature,
                             const std::vector<TransitionType>& transitions,
                             const std::vector<OpenSwath::SwathMap>& swath_maps,
-                            OpenSwath::SpectrumAccessPtr ms1_map,
+                            const OpenSwath::SpectrumAccessPtr& ms1_map,
```

Comment:
I think the general rule here is not to pass smartPtrs by const ref, but pass the underlying object as const ref instead.
But its an improvement, so let's leave it like it is for now.